src/app/GitExtensions.Extensibility/BrowseArguments.cs

```diff
@@ -0,0 +1,31 @@
+﻿using GitExtensions.Extensibility.Git;
+
+namespace GitExtensions.Extensibility;
```

Comment:
The namespace block should be kept in order to help git to get that the file has just been moved.

Reply:
Keeping it within the namespace doesn't help, the file is still marked as added/deleted.